cmd/notation/plugin.go

```diff
@@ -62,3 +108,169 @@ func listPlugins(command *cobra.Command) error {
 	}
 	return tw.Flush()
 }
+
+func installPlugin(command *cobra.Command, args []string, force bool) error {
+	if len(args) != 1 {
+		return errors.New("missing plugin package")
+	}
+
+	pluginPath := args[0]
+
+	var pluginBinary string
+	// if plugin contains a file path split and select the last element in the array
+	if strings.Contains(pluginPath, "/") {
+		pluginBinary = strings.Split(pluginPath, "/")[len(strings.Split(pluginPath, "/"))-1]
+	} else {
+		pluginBinary = pluginPath
+	}
+
+	// get plugin metadata
+	cmd := exec.Command("./"+pluginPath, "get-plugin-metadata")
```

Comment:
You can try [plugin.NewCLIPlugin](https://pkg.go.dev/github.com/notaryproject/notation-go/plugin#NewCLIPlugin) instead of manually call the `get-plugin-metadata` and parse the responses according the protocol.

Reply:
Got the changes made to implement this in a recent [commit](https://github.com/notaryproject/notation/pull/617/commits/e4d7ef5f0b4d53f2e1a218350742fab9f09c21ff). :)